dev-games/openscenegraph/openscenegraph-3.6.5-r101.ebuild

```diff
@@ -0,0 +1,162 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+LUA_COMPAT=( lua5-{1,2} )
+
+MY_PN="OpenSceneGraph"
+MY_P=${MY_PN}-${PV}
```

Comment:
See what I said about MY_COMMIT wrt pre-inherit pollution, usually would only have variables marked as `@PRE_INHERIT` in eclasses there (like LUA_COMPAT and WX_GTK_VER). Not that it's a problem per-se with MY_ namespace, but well.

Also most things I said about the other ebuild apply here, so won't repeat everything.